cr-examples/triton/src/main/java/oracle/code/triton/TritonTransformer.java

```diff
@@ -1053,7 +1053,7 @@ public Value dot(TensorType rType, Op.Result r,
             a = block.context().getValue(a);
             b = block.context().getValue(b);
             // Computed result is tensor of floats, regardless of inputs
-            Object zero = 0.0;
+            Object zero = 0.0f;
```

Comment:
The previous constant was `0.0` which parses as a `double` constant, not a `float` one. Of course the string representation of both would be the same -- but now that `AttributeWriter` appends the correct suffix, this issue led to weird mismatches in tests.